hypervisor/debug/profiling.c

```diff
@@ -171,11 +178,13 @@ static void profiling_disable_pmu(void)
 		__func__,  get_cpu_id());
 
 	if (ss != NULL) {
-		if (ss->vmexit_msr_cnt == 1) {
-			/* Set the VM Exit MSR Load in VMCS */
-			exec_vmwrite(VMX_EXIT_MSR_LOAD_COUNT, 0x0U);
-				exec_vmwrite64(VMX_GUEST_IA32_DEBUGCTL_FULL,
-				ss->saved_debugctl_value);
+		if (ss->vmexit_msr_cnt != 0) {
+			/* Restore the msr exit loading list of HV */
+			struct acrn_vcpu *vcpu = get_ever_run_vcpu(get_cpu_id());
+
+			exec_vmwrite64(VMX_EXIT_MSR_LOAD_ADDR_FULL,
+				hva2hpa(vcpu->arch.msr_area.host));
```

Comment:
Now we support 132 characters per line. Can you merge them into one line?

Reply:
Fixed